tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
     "files": [],
     "references": [
-        { "path": "src/server" },
-        { "path": "src/packages" }
```

Comment:
`server` indirectly pulls in all the `packages`.  If that weren't the case, I'd probably list them here individually - there's already a parallel list for the path mappings.

Reply:
The project this repo is based off of has some projects in packages that are not included in `server` - so this was convenient (vs manually adding every project that isn't in already in server).
Is the suggestion is to list them individually?

Reply:
Looking more into this, everything in `packages` is being pulled from either server or other projects at that level.

Reply:
You're going to list them all _somewhere_ - either here, in server, or in a dummy layer for the packages directory.  My point was mostly that that the `src/packages` project wasn't adding any value.